### ISSUE TITLE
Weird issue - disappearing entities

### ISSUE THREAD

**Issue by Dinth** [cgarwood/homeassistant-zwave_mqtt#97]
- Home Assistant version: 0.108.7
- ozwdaemon version or add-on version: 1.0.15 
- homeassistant-zwave_mqtt version: 0.0.10

Yesterday i have set up Zwave over MQTT integration and ozwdaemon on my fresh HA instance. 
All the devices on Zwave network were discovered and then i waited around 40 minutes to make sure that everything is properly initialised.
After that i started setting up devices (renaming them and assigning to rooms) through integrations page and i noticed a weird think with all my Fibaro Dimmers - they were not picking up current status. I could switch them on but their status was staying off and then i couldnt switch them off. There are already bug reports created for that: https://github.com/cgarwood/homeassistant-zwave_mqtt/issues/84 https://github.com/cgarwood/homeassistant-zwave_mqtt/issues/72
Other than that all the sensors were reporting their status and correctly populating entities.

I left the system idling overnight and this morning i found out that all Fibaro Dimmers suddenly started working correctly. 
But at the same time, all the sensors i have renamed yesterday stopped reporting anything and all the entities created by them are now "Unavailable". Other sensors (ones i havent touched yesterday) are still correctly populating entities.

**Comment by marcelveldt** [comment]
There is a new version coming up within a few days which should solve all these issues.
You will have to re-add the integration (as it will be a breaking change) but after that this should work like a breeze.

Please keep an eye on the new version!

**Comment by Dinth** [comment]
Thanks @marcelveldt for looking into that.
Are you talking about .11 version which just got released?

**Comment by marcelveldt** [comment]
Yes, but there is another fix coming up which did not make it into the .11 version.
So you can try .11 as it already fixes a lot of the issues but the next version will require you to re-do your setup as thing moved around a lot.

**Comment by Dinth** [comment]
Thanks. Is the next version still planned in a few days or because of .11 being released today, that has been pushed forward?

**Comment by marcelveldt** [comment]
Yeah, I think so. Give .11 a shot and see if that improves your situation.

**Comment by Dinth** [comment]
@marcelveldt  Unfortunately .11 has not fixed the issue with disappearing entities.
Actually, after updating to .11 i have removed and readded the integration and now after re-adding the integration, all the devices which i renamed day before yesterday are completely gone from the list.

**Comment by marcelveldt** [comment]
That is possible as we're still moving around a lot between id's. Remember that this is pre-beta so you can test it but not yet production ready. We're working towards a beta version which wil be more stable.

Please describe the behaviour you're seeing with version .11
The Dimmers not updating from time to time is a different issue though.

**Comment by Dinth** [comment]
The dimmers are actually working fine now.
The problem is that some of my devices (about 8, not only Fibaro Dimmers) are not visible in HA anymore. When i originally set up Zwave-over-MQTT all my devices were visible and i have renamed some of them - both names and entity names from within HA. Then, after few hours hours all the entity values which were renamed in HA (and only those) changed their state to  "Unavailable". After updating Zwave-over-MQTT to .11 nothing changed in this matter. I have tried removing integration and readding it, but now all the devices were added to HA except the ones i have previously renamed - those are now not even visible at all on Integrations config page. 
OZWDaemon log shows that all the devices are sending statistics and talking with it, so everything is working. 
So long story short, renaming devices and entities has broken them in Z-o-M and this persists somewhere even if the integration is removed, but i dont know where.

**Comment by marcelveldt** [comment]
Did you try restarting OZW ?

**Comment by Dinth** [comment]
Yes, i have tried restarting HA, OZW and host OS.

**Comment by marcelveldt** [comment]
Can you give me an example of entities that are missing ?
Also, did you check if the missing entities are just disabled and you can re enable them ?

If you want, can you send me a dump of your MQTT so I can do some exploring?
In Home Assistant developers tools, go to services and you can use the mqtt.dump service
It is enough to only dump the OpenZWave topic

**Comment by Dinth** [comment]
Please find the attached.
https://gofile.io/?c=UlStuA
Node 05 is an example of such problematic node.

**Comment by marcelveldt** [comment]
The issue is with your OZW daemon. I looked at your MQT dump and it's missing a lot of topics as if adding/interviewing those nodes is only half finished.
What you might want to try is simply press the connect/pair button on the device (sometimes you need to press it 3 times quickly in a row) so the device forcefully refreshes teh connection with the OZW host.

PROTIP: Get the tool "MQTT Explorer" and connect it to your mqtt broker. It will let you explore your entire Z-Wave mesh as MQTT topics.

And another screenshot from a healthy node

**Comment by Dinth** [comment]
@marcelveldt Hi Marcel, i hope you are well.
I havent tried refreshing devices forcefully yet, as its not as easy as it sounds - need electricians and scaffolding. 
But my Zwave mesh was recently rebuilt from scratch and OZW log was not showing any errors.
Surprisingly all devices which were gone from OZWdaemon for past few days suddenly appeared in Integration again yesterday evening. 
Ive noticed that the entities are disappearing straight after name change, so it cannot be an issue with Zwave network though. 

https://youtu.be/MDxYPx6rRCo
https://youtu.be/PbJma8ur2sM
